src/main/resources/application.properties

```diff
@@ -1,2 +1,23 @@
 uk.gov.companieshouse.chdorderapi.health=/chd-order-api/healthcheck
 uk.gov.companieshouse.chd.order.api.mid=/chd-order-api/missing-image-deliveries
+
+spring.datasource.url: ${CHPRD_DATASOURCE_URL}
+spring.datasource.username: ${CHPRD_SCHEMA_NAME}
+spring.datasource.password: ${CHPRD_PASSWORD}
+spring.datasource.driver.class: oracle.jdbc.driver.OracleDriver
+
+spring.jpa.hibernate.ddl-auto: none
+spring.jpa.show-sql: true
+spring.jpa.properties.hibernate.format_sql: true
+
+chprd.customer-id = CHPRD_CUSTOMER_ID
```

Comment:
Is this totally hardcoded then?

Reply:
Nice catch. This is incorrect. I have updated it now. 👍